### PR TITLE
Yelp query from slash command

### PR DESCRIPTION
**Pull request by kareemf**
Added ability to query yelp via slash command.
Using lunch-bot.send (slack incoming web hook) to send yelp lunch-bot yelp responses.
Bug fixes

### CODE REVIEW

File: src/lunch_bot/handler.clj [hunk 1]
```diff
@@ -24,7 +31,7 @@
     (cond
       (= com "random") (yelp/get-random)
       (= com "vote") (vote/start user text)
-      :else "What do you want?")))
+      :else (handle-query-request (body "text")))))
```
Comment (akoaysigod) [inline src/lunch_bot/handler.clj]: (= com "search") (handle-query-request (body "text")))
      :else "What do you want?")

File: src/lunch_bot/yelp.clj [hunk 2]
```diff
@@ -13,18 +14,37 @@
 (def ^:private zipcode "07302")
 
 (defn- parse-single-result [result]
-  (result :url))
+  (->
+    (str (result :name) " - " (first (first (result :categories))) " - " (result :url))
+    (send/send-response))
+    "done parse-single-result" result)
```
Comment (akoaysigod) [inline src/lunch_bot/yelp.clj]: (->
+    (str (result :name) " - " (first (first (result :categories))) " - " (result :url))
+    (send/send-response)))

File: src/lunch_bot/handler.clj (hunk 1 above)
Comment (kareemf) [inline src/lunch_bot/handler.clj]: Done

File: src/lunch_bot/yelp.clj (hunk 2 above)
Comment (kareemf) [inline src/lunch_bot/yelp.clj]: Removeing output string causes errors with buffer exiting prematurely. Ex, in curl
```
curl: (18) transfer closed with 20 bytes remaining to read
```

**Pull request merged**